static/app/components/organizations/pageFilters/container.spec.tsx

```diff
@@ -8,7 +8,7 @@ import OrganizationsStore from 'sentry/stores/organizationsStore';
 import OrganizationStore from 'sentry/stores/organizationStore';
 import PageFiltersStore from 'sentry/stores/pageFiltersStore';
 import ProjectsStore from 'sentry/stores/projectsStore';
-import {getItem, setItem} from 'sentry/utils/localStorage';
```

Comment:
cc @vuluongj20, there seemed to have been a lot of type errors in this file

Reply:
(Didn't write this file but) Thanks for fixing them!